controllers/falcon/falconnodesensor_controller.go

```diff
@@ -824,20 +817,28 @@ func (r *FalconNodeSensorReconciler) finalizeDaemonset(ctx context.Context, imag
 			}); err != nil {
 				return err
 			}
+			// Reset completedCount each loop, to ensure we don't count the same node(s) multiple times
+			var completedCount int32 = 0
```

Comment:
Technically this change isn't necessary since erroring usually results in starting over from the beginning of the function with completedCount already set to 0.